swervelib/rev/NeoSteerControllerFactoryBuilder.java

```diff
@@ -60,7 +60,7 @@ public FactoryImplementation(AbsoluteEncoderFactory<T> encoderFactory) {
 
         @Override
         public void addDashboardEntries(ShuffleboardContainer container, ControllerImplementation controller) {
-            // Logger.consoleLog("Called super.addDashboardEntries");
+            ConsoleLogger.consoleLog("Called super.addDashboardEntries");
```

Comment:
why adding this back in? here and below

Reply:
Actually below is just a rename - it's not adding it back. But why add it here?

Reply:
I'm not entirely sure why, I'll comment it out